app/views/admin/navigation_links/index.html.erb

```diff
@@ -2,26 +2,59 @@
   <div class="my-5"> Configure the left sidebar navigation links that appear at <a href="<%= app_url %>"><%= app_url %></a> using the form below:</div>
 
   <%= render "add_navigation_link_modal" %>
-  <% @navigation_links.each do |link| %>
-    <div class="flex pt-5 items-center">
-      <div class="mr-5"><%= link[:position] %></div>
-      <div class="admin-navigation-links">
-        <a href="<%= link[:url] %>" class="crayons-link crayons-link--block bg-base-20">
-          <div class="crayons-icon crayons-icon--default">
-              <%= link[:icon].html_safe %>
+  <div class="pt-8">
+    <div class="fw-bold">
+      <%= community_name %>
+    </div>
+    <% @default_nav_links.each_with_index do |link, index| %>
```

Comment:
accessibility enhancement (not blocking) - could we wrap these sets of links as a list with `ul`/`li` elements? It would help screen reader users as there's keyboard shortcuts to move between list items, and it would also announce how many are in the list etc.

It's not new to this PR, so if it's too much for now it could be a later enhancement 👍